packages/fcl-wc/src/fcl-wc.js

```diff
@@ -36,10 +31,22 @@ const initClient = async ({projectId, metadata}) => {
   }
 }
 
-export const initFclWc = async ({projectId, metadata} = {}) => {
+export const init = async ({
+  projectId,
+  metadata,
+  includeBaseWC = false,
+  sessionRequestHook = null,
+  wallets = [],
```

Comment:
Should we throw an error if wallets are passed in but flow.network is not mainnet?

Reply:
currently, wallet injection would only be possible on testnet